workflow/automation/submit/submit_empirical.py

```diff
@@ -22,16 +23,15 @@ def write_sl(sl_name, content):
         f.write(content)
 
 
-def generate_empirical_script(np, extended, cybershake_folder, realisations, out_dir):
-    # extended is '-e' or ''
+def generate_empirical_script(
+    np, extended_switch, cybershake_folder, realisations, out_dir
+):
+    # extended_switch is '-e' or ''
 
     faults = map(simulation_structure.get_fault_from_realisation, realisations)
     run_data = zip(realisations, faults)
-    run_data = [
-        (rel, fault)
-        for (rel, fault) in run_data
-        if rrup_file_exists(cybershake_folder, rel)
-    ]
+    run_data = [(rel, fault) for (rel, fault) in run_data]
+
     # determine NP
     # TODO: empirical are currently not parallel, update this when they are
     np = 1
```

Comment:
Just as a note this might need updating in the future once multiprocessing becomes a factor in empirical engine